sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/IncrementalExecution.scala

```diff
@@ -68,7 +71,7 @@ class IncrementalExecution(
    * Records the current id for a given stateful operator in the query plan as the `state`
    * preparation walks the query plan.
    */
-  private var operatorId = 0
+  private val operatorId = new AtomicInteger(0)
```

Comment:
not really. just a tiny bit cleaner to do `getAndIncrement` than using x and then calling `x += 1`